### DISCUSSION TITLE
"quoteProperties": "consistent"

### DISCUSSION THREAD

**Discussion by tylerbutler** [biomejs/biome#1974] [General]
Is there any reason that the quoteProperties config option doesn't support Prettier's `"consistent"` option? `"preserve"` is OK, but not as useful in my experience. Is this omitted intentionally? If it's mentioned in the "differences from prettier" docs, please let me know - I may have missed it.

**Comment by ematipico** [comment]
We don't track options on that page, so you didn't miss it.

We refrained from implementing this option because it can emit changes for lines of code that aren't modified by the user.

**Comment by phenomen** [comment]
I would like to use `consistent` too. This is my default go-to option in Prettier. The code looks ugly when half of the props in the same object have quotes and another half don't.
